lib/generators/sorcery/install_generator.rb

```diff
@@ -90,6 +90,12 @@ def only_submodules?
         options[:migrations] || options[:only_submodules]
       end
 
+	  def rails_version # for Rails5 ActiveRecord::Migration 
+		if    Rails::VERSION::MAJOR >= 5
+		  "[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
+		end
+	  end
+
```

Comment:
Same tabs issue here. Also does anything happen when rails major < 5? (I assume the function would return nil, and the rails_version hash just is ignored in rails <5.0?)

It might be more preferable to have this always return the Major/Minor, and then add || (Rails::VERSION::MAJOR < 5) instead. I think that would probably be better answered by a maintainer though.